### PR TITLE
Update credits for 0.F

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary
None

#### Purpose of change

Update credits for 0.F

#### Additional context

There are a lot of people who did great things for 0.F and I most definitely did not name everyone who's worthy, so suggestions are welcome.

#### Screenshots

**0.F credits (en)**:

image

**Legacy credits (en)**:

image

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
The screenshot is outdated?

**Comment by ZhilkinSerg** [comment]
> 
> 
> The screenshot is outdated?

Yeah, it is just for reference - I will update it eventually as suggestions go.

**Comment by jbytheway** [comment]
Can we keep the credits for previous versions somewhere that users can get at them?

File: data/credits/en.credits [hunk 1]
```diff
@@ -6,23 +6,22 @@
...
-<color_yellow>BrettDong</color> - For translations.
-<color_yellow>curstwist</color> - For roofs, mapgens, NPCs and changelogs.
-<color_yellow>davidpwbrown</color> - For auto-travel and auto-driving.  Also horse-riding.
-<color_yellow>jbytheway</color> - For CI work and making his (tri)point.
-<color_yellow>Hymore246</color> - For martial arts overhaul.
-<color_yellow>Qrox</color> - For readable backtraces in Windows builds.
-<color_yellow>ZhilkinSerg</color> - For mergetesting.
+<color_yellow>Aivean</color> - For outstanding performance.
+<color_yellow>anothersimulacrum </color> - For making me weary.
+<color_yellow>jbytheway</color> - For reducing Python flakiness, problem matching and consolidating blueprint requirements.
```
Comment (ZhilkinSerg) [inline data/credits/en.credits:15]:
```suggestion
<color_yellow>jbytheway</color> - For making me achieve things.
```

File: data/credits/en.credits [hunk 2]
```diff
@@ -6,23 +6,22 @@
...
-<color_yellow>BevapDin</color> - For code reviews and refactoring efforts.
-<color_yellow>BrettDong</color> - For translations.
-<color_yellow>curstwist</color> - For roofs, mapgens, NPCs and changelogs.
-<color_yellow>davidpwbrown</color> - For auto-travel and auto-driving.  Also horse-riding.
-<color_yellow>jbytheway</color> - For CI work and making his (tri)point.
-<color_yellow>Hymore246</color> - For martial arts overhaul.
-<color_yellow>Qrox</color> - For readable backtraces in Windows builds.
-<color_yellow>ZhilkinSerg</color> - For mergetesting.
+<color_yellow>Aivean</color> - For outstanding performance.
+<color_yellow>anothersimulacrum </color> - For making me weary.
```
Comment (jbytheway) [inline data/credits/en.credits:14]:
```suggestion
<color_yellow>anothersimulacrum</color> - For making me weary.
```
Removing stray space.

**Comment by ZhilkinSerg** [comment]
> Can we keep the credits for previous versions somewhere that users can get at them?

I am not against that, but IIRC @kevingranade wanted to show credits only for current version specifically.

**Comment by kevingranade** [comment]
I'm guessing they mean just stash the file somewhere?

**Comment by jbytheway** [comment]
I was thinking keep the old files around and maybe also have a way to tab through the files in the credits screen.

**Comment by kevingranade** [comment]
Yea tabbing through to older versions would be fine and I doubt it'd be hard, I was just against making the main window just keep expanding.

**Comment by ZhilkinSerg** [comment]
Credits window already has fixed size and is scrollable. I can restore credits for older version and we can implement tabs or something similar later.

**Pull request merged**